DataConnectors/Okta Single Sign-On/azuredeploy_OktaSingleSignOn_API_FunctionApp_V2.json

```diff
@@ -193,7 +195,11 @@
                         },
                         {
                             "name": "WEBSITE_RUN_FROM_PACKAGE",
-                            "value": "https://aka.ms/sentineloktaazurefunctioncodev2"
+                            "value": "https://github.com/v-jayakal/Azure-Sentinel/blob/v-jayakal/oktaGovTest/DataConnectors/Okta%20Single%20Sign-On/AzureFunctionOktaSSO_V2/AzureFunctionOktaSSO.zip?raw=true"
```

Comment:
just a note, we need to modify this back after customer tested with the private change.